src/app/components/Metadata/index.tsx

```diff
@@ -82,17 +82,30 @@ const MetadataContainer = ({
   } = useTheme();
   const appleTouchIcon = getAppleTouchUrl(service);
   const isEnglishService = ENGLISH_SERVICES.includes(service);
+  const pathsForUkLink = [
+    '/sport/formula1',
+    '/sport/cricket/articles',
+    '/sport/rugby_union/articles',
+    '/sport/rugby_league/articles',
+  ];
+
+  const isUKSport = pathsForUkLink.some(
```

Comment:
I know that these are sport links for now, but I wonder if we could make this variable more generic, so that it still makes sense if non-sport links are added? 

```suggestion
  const isUKPath = pathsForUkLink.some(
```

or 

```suggestion
  const isUKLink = pathsForUkLink.some(
```